src/components/AssetInput/SingleAsset/SelectedAsset.tsx

```diff
@@ -39,13 +39,17 @@ export const SelectedAsset: FC<Required<SelectedAssetProps>> = ({ asset, size, a
 
     const [flyoutWidth, setFlyoutWidth] = useState(0);
 
-    const calcFlyoutWidth = () => {
-        setFlyoutWidth(buttonRef.current?.getBoundingClientRect().width || 0);
-    };
-
     useEffect(() => {
-        calcFlyoutWidth();
-        window.addEventListener("resize", calcFlyoutWidth, false);
+        const calcFlyoutWidth = () => {
+            setFlyoutWidth(buttonRef.current?.getBoundingClientRect().width || 0);
+        };
```

Comment:
```suggestion
        const calculateFlyoutWidth = () => {
            setFlyoutWidth(buttonRef.current?.getBoundingClientRect().width ?? 0);
        };
```
We have a preference for `??` over `||` in the codebase.
I would also rename the method to a more verbose one (we anyway minify everything)